build/dockerfiles/Dockerfile

```diff
@@ -31,6 +31,7 @@ FROM registry.centos.org/centos/httpd-24-centos7 AS registry
 RUN mkdir -m 777 /var/www/html/devfiles
 COPY .htaccess README.md /var/www/html/
 COPY --from=builder /build/devfiles /var/www/html/devfiles
+COPY ./images /var/www/html/images
```

Comment:
do we need to set 777 perms here too?

Reply:
It shouldn't be necessary, since the files are by default group-readable and we're not looking to modify them. I suppose it could be done for safety, though.